### DISCUSSION TITLE
Bonemerang

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#533] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/bonemerang/

**Comment by angellisy** [comment]
E dizem que o Mewtwo é do mal

**Comment by najarin-the-loremaster** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/imagens/tirinhas/bonemerang/#comment_1114016647>):

> E dizem que o Mewtwo é do mal

Esse é um tema bastante controverso.



Pessoalmente, eu não vejo ele como mau e sim confuso e incompreendido. Ele foi criado por cientistas que cometeram atrocidades terríveis apenas pra satisfazer os caprichos de um grupo de criminosos. Sem nenhuma educação ou direcionamento, ele se tornou violento, mau, rancoroso e só graças a um enorme sacrifício ele pôde ver a humanidade com outros olhos.
